### DISCUSSION TITLE
Alternative approaches to integrating Capacitor into an existing Laravel/Vue project without index.html

### DISCUSSION THREAD

**Discussion by milosevicn** [ionic-team/capacitor#7058] [Q&A]
I'm currently working on integrating Capacitor into an existing Laravel/Vue project. However, due to the nature of Laravel applications, the project doesn't have an `index.html` file, which is typically required by Capacitor during the initialization process.

I'm wondering if there are any alternative approaches or workarounds to implement Capacitor into such a project without creating a separate `index.html` file.

I'd appreciate any insights or suggestions on how to proceed with this integration without compromising the existing Laravel/Vue structure.

**Comment by GeoffreyOlivier** [comment]
I'm interrested too
